src/ja/server/proxy/command_handler.py

```diff
@@ -60,9 +63,12 @@ def _user_is_admin(self, user: str) -> bool:
         return self._admin_group in groups
 
     def _execute_command(self, command: ServerCommand) -> Dict[str, object]:
+        logger.info("executing server command")
+        logger.debug("%s" % str(command))
         return command.execute(self._database).to_dict()
```

Comment:
Wouldn't it be good to log the output of the command execution like you you did in `WokerProxy`? Or maybe specify if it succeeded or not.

Reply:
Maybe, but I tried to add the logging stuff without formatting the existed code. But maybe I can change it a bit.

Reply:
I addressed this in my latest fix.